Gemfile

```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
 gem 'inifile'
-gem 'azure_mgmt_compute', '~> 0.11'
-gem 'azure_mgmt_resources', '~> 0.11'
-gem 'azure_mgmt_network', '~> 0.11'
+gem 'azure_mgmt_compute', '~> 0.5', '= 0.14.0'
```

Comment:
If you have an equality pin, the pessimistic pin `~> 0.5` is unnecessary. I'd remove these to be abundantly clear what you're looking for.